### ISSUE TITLE
Scaled box coordinates when computing loss in test.py

### ISSUE THREAD

**Issue by elmeriz** [ultralytics/yolov5#1528]
compute_loss() in test.py uses box coordinates that are scaled to image size. This results in zero cls/box losses.
In train.py, loss is computed with normalized coordinates which works as expected.

In test.py (line 103):

        targets = targets.to(device)
        nb, _, height, width = img.shape  # batch size, channels, height, width
        targets[:, 2:] *= torch.Tensor([width, height, width, height]).to(device)

        with torch.no_grad():
            # Run model
            t = time_synchronized()
            inf_out, train_out = model(img, augment=augment)  # inference and training outputs
            t0 += time_synchronized() - t

            # Compute loss
            if training:
                loss += compute_loss([x.float() for x in train_out], targets, model)[1][:3]  # box, obj, cls
        ...

Simple fix is to just scale the coordinates after computing loss:

        targets = targets.to(device)
        nb, _, height, width = img.shape  # batch size, channels, height, width

        with torch.no_grad():
            # Run model
            t = time_synchronized()
            inf_out, train_out = model(img, augment=augment)  # inference and training outputs
            t0 += time_synchronized() - t

            # Compute loss
            if training:
                loss += compute_loss([x.float() for x in train_out], targets, model)[1][:3]  # box, obj, cls

            targets[:, 2:] *= torch.Tensor([width, height, width, height]).to(device)
        ...

**Comment by glenn-jocher** [comment]
@elmeriz ah, thanks for the bug report! This seems like an easy fix, I like your suggestion.

**Comment by glenn-jocher** [comment]
@elmeriz your proposed fix has been implemented and merged in PR #1529. Please git pull to receive this update and let us know if you have any other problems. 

Thank you for your contributions, and also feel free to submit PRs for any other fixes and updates you find!
